### PR TITLE
if on sell tab, show sell price and if on buy tab show buy price

### PR DESCRIPTION
**Pull request by pimato**

### CODE REVIEW

File: src/services/contractSimulators/cOrgSim.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { BigNumber } from '../../utils/bignumber';
 import { bnum } from '../../utils/helpers';
 import BigDivSim from './bigDivSim';
 import { DatState } from '../../stores/datStore';
+import { formatBalance } from 'utils/token';
```
Comment (AugustoL) [inline src/services/contractSimulators/cOrgSim.ts:5]: this is imported but not used.

File: src/components/BondingCurveChart/BondingCurveChart.tsx [hunk 2]
```diff
@@ -141,7 +141,11 @@ const BondingCurveChart = observer((totalSupplyWithoutPremint:BigNumber) => {
             }
     
         totalSupplyWithoutPremint = totalSupplyWithPremint.minus(initReserve);
-        currentSellPrice = reserveBalance.times(reserveBalance).div(totalSupplyWithPremint);
+        currentSellPrice = reserveBalance.times(2).div(totalSupplyWithPremint);
+        console.log(formatBalance(reserveBalance));
```
Comment (AugustoL) [inline src/components/BondingCurveChart/BondingCurveChart.tsx:145]: console.logs heree

Comment (pimato) [reply src/components/BondingCurveChart/BondingCurveChart.tsx]: All removed

File: src/services/contractSimulators/cOrgSim.ts (hunk 1 above)
Comment (pimato) [reply src/services/contractSimulators/cOrgSim.ts]: removed

**Pull request merged**